qutip/tests/core/test_ptrace.py

```diff
@@ -55,7 +55,7 @@ def expected(qobj, sel):
             continue
         tmp_dims = (before, dim, after) * 2
         out = np.einsum('aibcid->abcd', out.reshape(tmp_dims))
-    return qutip.Qobj(out.reshape(new_shape), dims=dims)
+    return qutip.Qobj(out.reshape(new_shape), dims=dims, type='oper')
```

Comment:
It seems that Qobj is not able to infer the type from an empty dims (dims=[[][]]).

Reply:
That seems not completely unexpected.

Reply:
I was not very clear what the problem was from the error message so it took me a little bit of digging to figure it out.